practice-typescript/src/styles/modules/message.scss

```diff
@@ -23,4 +23,12 @@
     color: $message-error;
     background-color: $bg-error;
   }
+}
+
+.emty-table {
```

Comment:
emty?

Reply:
fixed , I changed emty-table to empty-table